Retrospring/retrospring
# Refactor reactions into appendables

**Things to test:**

- Before checking out this branch, create some answers and comments and smile them. 
- Check out this branch and run the migration 
- Check that the answer and comment smiles have migrated to apppendables
- Test adding and removing a smile on an answer
- Test adding and removing a smile on an answer
- Test that notifications get sent correctly 

Closes #346 
Closes #347
